components/datetime/src/format/datetime.rs

```diff
@@ -573,6 +573,112 @@ impl RequiredData {
     }
 }
 
+/// What data is required to format a given pattern.
+#[derive(Default)]
+pub(crate) struct RequiredDataNeo {
+    // C::YearSymbolsV1Marker is required.
+    pub year_symbols: Option<Field>,
```

Comment:
issue: we should document the invariant that the field must be a year field (etc)

Reply:
Already fixed in f53809a